Rakefile

```diff
@@ -1,8 +1,6 @@
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 
-RSpec::Core::RakeTask.new(:spec) do |task|
-  task.ruby_opts = "-W"
```

Comment:
IIRC I actually removed it, you defended its presence, and then later converted the code base to use rspec and removed it at which point I proposed keeping it... kind of a long and complicated relationship.... anyway I'm OK with turning it off because most of the compiler warnings are also rubocop cops which lets us display them in cc which is nice. The only reason I would want to keep them on at this point is because this is a gem and gems shouldn't generate warnings in case their consumers have warnings turned on. But I think this is only a gem for convenience, so it's less important. Also that warning is annoying.